README.md

```diff
@@ -1 +1,3 @@
-# workshop
\ No newline at end of file
+# workshop
+
+- This project is a pilot to describe development wo[f]kflow.
```

Comment:
correct typo on `workflow.`

Reply:
thanks for your review, i will change it.